xbmc/cores/VideoPlayer/DVDCodecs/DVDFactoryCodec.cpp

```diff
@@ -55,73 +44,16 @@
 #include "settings/VideoSettings.h"
 #include "utils/StringUtils.h"
 
-CDVDVideoCodec* CDVDFactoryCodec::OpenCodec(CDVDVideoCodec* pCodec, CDVDStreamInfo &hints, CDVDCodecOptions &options )
-{
-  try
-  {
-    CLog::Log(LOGDEBUG, "FactoryCodec - Video: %s - Opening", pCodec->GetName());
-    if( pCodec->Open( hints, options ) )
-    {
-      CLog::Log(LOGDEBUG, "FactoryCodec - Video: %s - Opened", pCodec->GetName());
-      return pCodec;
-    }
 
-    CLog::Log(LOGDEBUG, "FactoryCodec - Video: %s - Failed", pCodec->GetName());
-    delete pCodec;
-  }
-  catch(...)
-  {
-    CLog::Log(LOGERROR, "FactoryCodec - Video: Failed with exception");
-  }
-  return nullptr;
-}
-
-CDVDAudioCodec* CDVDFactoryCodec::OpenCodec(CDVDAudioCodec* pCodec, CDVDStreamInfo &hints, CDVDCodecOptions &options )
-{
-  try
-  {
-    CLog::Log(LOGDEBUG, "FactoryCodec - Audio: %s - Opening", pCodec->GetName());
-    if( pCodec->Open( hints, options ) )
-    {
-      CLog::Log(LOGDEBUG, "FactoryCodec - Audio: %s - Opened", pCodec->GetName());
-      return pCodec;
-    }
-
-    CLog::Log(LOGDEBUG, "FactoryCodec - Audio: %s - Failed", pCodec->GetName());
-    delete pCodec;
-  }
-  catch(...)
-  {
-    CLog::Log(LOGERROR, "FactoryCodec - Audio: Failed with exception");
-  }
-  return nullptr;
-}
-
-CDVDOverlayCodec* CDVDFactoryCodec::OpenCodec(CDVDOverlayCodec* pCodec, CDVDStreamInfo &hints, CDVDCodecOptions &options )
-{
-  try
-  {
-    CLog::Log(LOGDEBUG, "FactoryCodec - Overlay: %s - Opening", pCodec->GetName());
-    if( pCodec->Open( hints, options ) )
-    {
-      CLog::Log(LOGDEBUG, "FactoryCodec - Overlay: %s - Opened", pCodec->GetName());
-      return pCodec;
-    }
-
-    CLog::Log(LOGDEBUG, "FactoryCodec - Overlay: %s - Failed", pCodec->GetName());
-    delete pCodec;
-  }
-  catch(...)
-  {
-    CLog::Log(LOGERROR, "FactoryCodec - Audio: Failed with exception");
-  }
-  return nullptr;
-}
+//#include "VPFactoryCodec_override.h"
```

Comment:
yes, it is planned to move the platform dependent code to separate files. waiting for @fetzerch to complete the cmake part.